### ISSUE TITLE
Chat with GM from orther faction

### ISSUE THREAD

**Issue by Winchester1987** [TrinityCore/TrinityCore#22271]
I have everything default.
GM can whisper to anyone.
Horde player cannot whisper to alliance GM
Alliance player cannot whisper to horde GM

TDB: 335.64
Last commit: e5b8b1e99a4e7361be862ae6e1f29189907167e4
(Branch 3.3.5a, Debian 9, Release)

**Comment by Aokromes** [comment]
.gm on and use gm chat system.

**Comment by Winchester1987** [comment]
GM mode is on
Gm chat badge is on
The player cannot respond even in the GM chat window

**Comment by shamage** [comment]
I do not understand . Can not you use the GM master feature 
using the `.gm on` command ?

**Comment by shamage** [comment]
I suggest you perform a new clean compilation and test the problematic features you mentioned. Maybe these problems will disappear with a new compilation.

**Comment by Winchester1987** [comment]
I will do that.
The only thing I discovered is that the problem appears after re running wordserver.exe

**Comment by Winchester1987** [comment]
I remade everything.
I created 2 accounts, a GM one and a Player one.
The issue is the same.
A player cannot whisper to the first GM char created, who is from the other faction.
If I create a second GM char, a player can whisper him.
If I shut down and re run worldserver.exe, the second GM char has the same problem.

Is anyone having the same problem?

**Comment by Aokromes** [comment]
works for me with a 9 years old gm char and a lot of years old character:
https://i.imgur.com/vZPCA3Y.png

**Comment by Winchester1987** [comment]
Thank you for testing the problem.
I don´t know what could be the problem in my case.
I have tested this on linux, on windows.
I have truncated and created the rbac_linked_permissions running worldserver.exe to see if the problem could be related to it.
I have deleted and created chars, accounts, using all GM ranks.
I don´t know what else I can do. Where could I check to see if I can fix the problem?

It seemsthat  the problem is just for me. Maybe I should go to the forum instead of replying to this since it is not a real issue.
